test/sqllogictest/attributes/mir_arity.slt

```diff
@@ -152,7 +152,7 @@ Explained Query:
               Filter (#1 = 1) // { arity: 2 }
                 Get l0 // { arity: 2 }
             Project (#0) // { arity: 1 }
-              Filter error("more than one record produced in subquery") AND (#1 > 1) // { arity: 2 }
```

Comment:
This would be prevented by https://github.com/MaterializeInc/materialize/pull/30171, right?